### ISSUE TITLE
macOS 12.2, --sudoers file content is not working

### ISSUE THREAD

**Issue by vikingtoby** [sshuttle/sshuttle#723]
Even moving past issue: https://github.com/sshuttle/sshuttle/issues/722 reverting the other changes.

Sudo-less sshuttle doesn't seem to be working with the `/etc/sudoers.d/sshuttle-auto` it generates:
```
# WARNING: When you allow a user to run sshuttle as root,
# they can then use sshuttle's --ssh-cmd option to run any
# command as root.

Cmnd_Alias SSHUTTLE842 = /usr/bin/env PYTHONPATH=/usr/local/Cellar/sshuttle/1.1.0/libexec/lib/python3.10/site-packages /usr/local/Cellar/sshuttle/1.1.0/libexec/bin/python3.10 /usr/local/bin/sshuttle *

<username> ALL=NOPASSWD: SSHUTTLE842
```

It still requires sudo permissions in an unauthenticated session.

**Comment by vincent-dm** [comment]
Same here (MacOS `12.2`, sshuttle `1.1.0`):

```
~ sshuttle --sudoers
# WARNING: When you allow a user to run sshuttle as root,
# they can then use sshuttle's --ssh-cmd option to run any
# command as root.
Invalid sudoers filename: Final sudoers file location (/private/etc/sudoers.d/sshuttle_auto) does not begin with /etc/sudoers.d
Failed updating sudoers file.
```

**Comment by skuhl** [comment]
This should be fixed by merge #727.
